torch/ao/quantization/observer.py

```diff
@@ -1405,3 +1443,15 @@ def load_observer_state_dict(mod, obs_dict):
 """
 Default observer for a floating point zero-point and 4 bit activations.
 """
+
+# TODO(future PR): remove these defaults and enforce activation functions
+# to explicitly specify their output range
+default_symmetric_fixed_qparams_args =\
```

Comment:
good idea, maybe we can pass in this in the register_pattern and use this to initialize both FakeQuantize and FixedQParamObserver constructors?